ERCS/erc-7635.md

```diff
@@ -0,0 +1,521 @@
+---
+eip: 7635
+title: Multi-Fungible Token
+description: A new transfer model and an approval model for storing various types of assets.
+author: Luiswll Wang (@wluisw) <luiswll@memelabs.dev>, william <memeking@memelabs.dev>, yaron <yaron@memelabs.dev>
+discussions-to: https://ethereum-magicians.org/t/erc-7635-the-multi-fungible-token/18906
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-02-22
+requires: 165, 721
+---
+
+## Abstract
+
+This is the standard for Multi-Fungible Tokens (MFT). The provided document describes the smart contract interface set
+that defines the [ERC-721](./eip-721.md) compatible token standard. It also introduces a new transfer model and an approval model for
+storing various types of assets.
+
+MFT tokens include an [ERC-721](./eip-721.md) equivalent ID property to identify themselves as unique entities, enabling MFT tokens to
+be transferred between addresses and operated upon in an [ERC-721](./eip-721.md) compatible manner.
+
+MFT implements multiple slots to configure assets of types such as [ERC-721](./eip-721.md), [ERC-20](./eip-20.md), KEY, etc. Its core features include:
+
+* MFT Management
+  * Flexible configuration of MFT transfer permissions, MFT types, and levels.
+* Slot Management
+  * Flexible configuration of slot asset types, asset contract addresses, and transfer permissions.
+* Asset Management
+  * Through slots, support storage and configuration of [ERC-721](./eip-721.md), [ERC-20](./eip-20.md), and KEY assets.
+  * Enable asset transfer between MFTs.
+  * Enable asset transfer between MFTs and EOA wallets.
+  * Flexible deposit and withdrawal of tokens in MFT.
+
+Compatibility and Scalability
+Fully compatible with the [ERC-721](./eip-721.md) protocol, allowing free trading on decentralized exchanges.
+This EIP introduces a new token model to achieve token multi-dimensionality, including the ability for the same MFT to
+configure storage for multiple types of token assets, as well as value transfers between two assets within the same slot
+and value transfers from tokens to addresses.
+
+## Motivation
+
+The essence of tokenization is how assets are encapsulated. Traditionally, there have been two methods used: fungible
+tokens ([ERC-20](./eip-20.md)) and non-fungible tokens ([ERC-721](./eip-721.md)). These correspond to homogeneous assets and unique assets,
+respectively. The former is similar to stocks or currencies, while the latter is similar to art collections or game
+items.
+
+However, there is an increasing need for encapsulating a wide variety of complex assets. The existing protocol standards
+do not fully meet the requirements for encapsulating such assets.
+
+For example, an account may hold multiple fungible assets such as USD, GBP, JPY, along with a credit account, insurance
+or investment contracts, and loyalty points（KEY） from a bank. Alternatively, a game character may possess fungible
```

Comment:
Is there a reason to use [fullwidth left parenthesis](https://www.fileformat.info/info/unicode/char/ff08/index.htm) instead of just a regular one?